vochain/scrutinizer/indexertypes/types.go

```diff
@@ -212,6 +212,7 @@ type VotePackage struct {
 type VoteReference struct {
 	Nullifier    types.HexBytes `badgerholdKey:"Nullifier"`
 	ProcessID    types.HexBytes `badgerholdIndex:"ProcessID"`
+	VoterID      types.HexBytes
```

Comment:
Do we know why VoteReference and EnvelopeMetadata are almost duplicated types? Its quite weird.

VoterID should be stored only once in the correct place.

Reply:
VoteReference is for the database itself and is required for fetching db info, EnvelopeMetadata is used for the envelope list API so voterID should be included in both as long as we use the same approach of having different types for the db and the API. Also for generating sqlc code voterID must be added into the VoteReference.